Grid events add export and column events to main article (#841)
telerik/blazor-docs: components/grid/events.md

Review the [grid state]({%slug grid-state%}) article for more details and examples on how the grid state works and what you can do with it.

### Column Events

> The grid column emits the `OnCellRender` event when rendering the cells so you can customize them. You can read more and find examples in the [Grid Column Events]({%slug grid-column-events%}) article.

Reviewer: ```suggestion
The Grid columns emit the `OnCellRender` event, so you can customize each cell separately. Read more and find examples in the [Grid Column Events]({%slug grid-column-events%}) article.
```

### Command Button Click

The command buttons of a grid provide an `OnClick` event before firing their built-in command (such as opening a row for editing, or adding a new row). You can do this to implement some additional logic and to also handle custom commands - both from a [Command Column]({%slug components/grid/columns/command%}), and from a [Toolbar Button]({%slug components/grid/features/toolbar%})

### Export Events

> When exporting the content of the grid, it will fire events like `OnBeforeExport` and `OnAfterExport`. They allow you to get or modify the columns and their settings, provide custom data, to cancel the export, and to also get the final file output. You can read more about them and find code examples in the [Grid Export Events]({%slug grid-export-events%}) article.

Reviewer: ```suggestion
During export, the Grid will fire events like `OnBeforeExport` and `OnAfterExport`. They allow you to:

* Get or modify the columns and their settings;
* Provide custom data;
* Cancel the export;
* Get the final file output;

Read more about them and find code examples in the [Grid Export Events]({%slug grid-export-events%}) article.
```
